### ISSUE TITLE
WebGL - WebAssembely not supporte   

### ISSUE THREAD

**Issue by bargmy** [nwjs/nw.js#8220]
i have the 0.50.4 version of NW.JS Arm build on my raspberry pi.
the problem is when i redirect to a html game ( url ) that requires web assembely or webGL... it just errors ; Webgl is not supported update your browser; 
but when i run the game from package.nw it work fine without error... 

"chromium-args": "--enable-webgl --enable-webgl2 --enable-webgl2-compute-context --ignore-gpu-blacklist --enable-experimental-canvas-features --disable-plugins --disable-internal-flash --disable-popup-blocking --enable-gamepad --enable-html5-camera --allow-file-access-from-files"

Device : Raspberry PI 5 4gB ram    --  Raspbian Lite 32bit      --     xfce

**Comment by ayushmanchhabra** [comment]
You have a HTML game deployed on a website. You also have the source code with you. When you run the game as a website in NW.js, it gives a WebGL is not supported error. When you package the game's source inside the package.nw and run in with NW.js, it works fine. Is this correct?

**Comment by bargmy** [comment]
> You have a HTML game deployed on a website. You also have the source code with you. When you run the game as a website in NW.js, it gives a WebGL is not supported error. When you place the game's source inside the package.nw and run in with NW.js, it works fine. Is this correct?

Yes exactly. sorry for responding lately.. also the game im deploying its on localhost.

**Comment by ayushmanchhabra** [comment]
If you want to run the game as a website in NW.js, then you need to pass the same flags you have defined in chromium-args to the nw executable. For example:

```shell
nw --flag=value --another-flag=another_value
```

**Comment by bargmy** [comment]
> If you want to run the game as a website in NW.js, then you need to pass the same flags you have defined in chromium-args to the nw executable. For example:
> 
> ```shell
> nw --flag=value --another-flag=another_value
> ```

okay so i did but i still had the issue.. 
 but intresting thing is chrome://gpu says webgl2 & webgl is enabled! 
 
![image](https://github.com/user-attachments/assets/34ce8a4e-7d34-406d-a2be-d521a4a840e8)
but any unity game or 3D game wont run
![image](https://github.com/user-attachments/assets/c9bd3dd6-4d5a-4d9a-a231-819a164247e4)

i tried some emulators but only Sega Genesis & GameBoy didnt crash because of any webgl or something errors

**Comment by ayushmanchhabra** [comment]
This sounds like a bug..

**Comment by bargmy** [comment]
> This sounds like a bug..

should i use opengl mesa or something?

**Comment by glacios** [comment]
@bargmy 
Have you tried `--ignore-gpu-blocklist` (bl**o**cklist, not bl**a**cklist) chromium argument?  
Also use `Copy report to clipboard` button instead of screenshoting part of report

(migth be related to #8192)

UPD: It might be fixed in v0.101.0 (no extra arguments needed)

**Comment by GnorTech** [comment]
This was likely fixed in a newer version. Please try 0.111.0 and reopen if the issue persists.
